lib/hooks/usePoolChainValues.js

```diff
@@ -97,6 +118,21 @@ export const fetchPoolChainValues = async (
 
       const values = await batch(provider, ...batchRequests)
 
+      let sablierStream = null
+      if (beforeAwardListener) {
+        sablierStream = await fetchSablierStreamChainValues(
+          provider,
+          chainId,
+          beforeAwardListener,
+          prizePool,
+          {
+            prizePeriodStartedAt: values.prizeStrategy.prizePeriodStartedAt[0],
```

Comment:
Since we use these again below on lines 150-152 let's pull these out into their own vars so we can just pass `{ prizePeriodStartedAt, prizePeriodSeconds, ... }`